de.metas.adempiere.adempiere/base/src/main/java/de/metas/banking/api/IBPBankAccountDAO.java

```diff
@@ -10,31 +10,31 @@
  * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 2 of the
  * License, or (at your option) any later version.
- * 
+ *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
  * GNU General Public License for more details.
- * 
+ *
  * You should have received a copy of the GNU General Public
  * License along with this program. If not, see
  * <http://www.gnu.org/licenses/gpl-2.0.html>.
  * #L%
  */
 
 import java.util.List;
+import java.util.Optional;
 import java.util.Properties;
 
+import de.metas.bpartner.BPartnerId;
+import de.metas.money.CurrencyId;
+import lombok.NonNull;
 import org.compiere.model.I_C_BP_BankAccount;
 
 import de.metas.util.ISingletonService;
 
 /**
- * 
- * This interface must be moved in de.metas.banking after we get rid of org.compiere.model.MPaySelectionCheck !
- * 
- * @author metas-dev <dev@metasfresh.com>
- *
+ * todo This interface must be moved in de.metas.banking but there still are some dependencies which need be fixed
```

Comment:
```suggestion
 * TODO This interface must be moved in de.metas.banking but there still are some dependencies which need be fixed
```
..upercase TODO pls...that way it's noticed by eclipse

Reply:
ohhhhhhh, i never has this problem in intellij. for me its case insensitive. i can just do `// tODo bla` and it would still show up.